repos/roslyn.proj

```diff
@@ -1,5 +1,25 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))/dir.props" />
```

Comment:
Seems like a lot of ceremony here. Should be able to simplify to 

``` 
<Import Project="dirs.props" />
```

Reply:
Yes but it is copy and past-able in all projects and originally I didn't have a dir.props file. It is a nice convention that we use everywhere.

Reply:
Both are copy pastable. There is no semantic difference between the two.

Reply:
except one requires a dir.props in every directory and the other doesn't as it will find the next one in the directory structure, so if we move this file around or move them into there own subdirectories or things like that we don't need to change the files at all, which is why I prefer the approach I used.